packages/plugin-lighthouse/src/lib/utils.unit.test.ts

```diff
@@ -54,3 +59,265 @@ describe('validateOnlyAudits', () => {
     ).toThrow(new AuditsNotImplementedError(['d']));
   });
 });
+
+describe('toAuditOutputs', () => {
+  it('should parse valid lhr details', () => {
+    expect(
+      toAuditOutputs({
+        audit: {
+          scoreDisplayMode: 'binary',
+          title: 'audit',
+          description: 'audit',
+          score: null,
+          id: 'audit',
+        },
+      }),
+    ).toStrictEqual([
+      {
+        displayValue: undefined,
+        score: 1,
+        slug: 'audit',
+        value: 0,
+      },
+    ]);
+  });
+
+  it('should not parse lhr details debugdata', () => {
```

Comment:
The tests should verify the currently implemented logic which is that a lot of the types are not supported yet and some are not implemented yet.

So the same description edit should go to all other types that are not implemented yet.

```suggestion
  it('should inform that debugdata type is not supported yet', () => {
```